rust/src/lib/ip.rs

```diff
@@ -299,6 +309,18 @@ impl InterfaceIpv4 {
             }
         }
 
+        if let Some(forwarding) = self.forwarding {
+            if !self.enabled && forwarding {
```

Comment:
Maybe just `if !self.enabled`? It neither makes much sense to define `false`, doesn't it?

Reply:
Since the corresponding IP family is disabled, it’s appropriate to define forwarding: false for the device. This makes the configuration explicit, though omitting the field is also valid since forwarding cannot be enabled while the family is disabled.